db/data/regions/AC.yml

```diff
@@ -18,5 +18,10 @@ format:
   show: "{firstName} {lastName}_{company}_{address1}_{address2}_{city}_{zip}_{country}_{phone}"
 emoji: 🇦🇨
 languages:
-  - en
+- en
```

Comment:
Note the language indentation was corrected as a result of the script to bring over example addresses